packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -236,8 +247,9 @@ export class FabricTestLedgerV1 implements ITestLedger {
       const privatePort = 7050;
       const hostPort = await Containers.getPublicPort(privatePort, cInfo);
       const url = `grpcs://localhost:${hostPort}`;
-      const ordererPemPath =
-        "/fabric-samples/first-network/crypto-config/ordererOrganizations/example.com/tlsca/tlsca.example.com-cert.pem";
+      const ordererPemPath = this.getFabricVersion().valueOf().includes("1.4.")
```

Comment:
All 3 of my comments apply here as well regarding the version checks, the `valueOf()` and the magic strings.

Reply:
Incorporated the changes. Please have a re-look.